source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -386,6 +410,10 @@ Parameter Details
     - Type
     - Value
 
+  * - asr_phrases
+    - Array of hashes
+    - A list of phrases that apply to the child of a Menu node. Can only be used when speech recognition is enabled. Allows the caller to respond verbally with one of the configured phrases instead of making a keypress. For example, the first child of a Menu node may have a value of [{"phrase": "sales"}, {"phrase": "support"}] for "asr_phrases", where the caller may say "sales" or "support" to select the Menu option instead of pressing 1.
```

Comment:
Should we also specify that it can have the prompt recording id here as well?

Reply:
Given that we currently don't support setting these prompts manually via the UI, I'd suggest we not do this and wait until we've heard feedback from customers indicating that it's something they would want. (and if we do, we'll then build it fully into the UI and the API)